go/internal/feast/onlinestore.go

```diff
@@ -33,7 +34,7 @@ type OnlineStore interface {
 	// Feature object as pointers in GetOnlineFeaturesResponse)
 	// => allocate memory for each field once in OnlineRead
 	// and reuse them in GetOnlineFeaturesResponse?
-	OnlineRead(ctx context.Context, entityKeys []types.EntityKey, featureViewNames []string, featureNames []string) ([][]FeatureData, error)
+	OnlineRead(ctx context.Context, entityKeys []*types.EntityKey, featureViewNames []string, featureNames []string) ([][]FeatureData, error)
```

Comment:
Why are we changing the interface here?

Reply:
I think it doesn't make sense to have to copy the EntityKey around, changing it to a pointer doesn't change the functionality, just makes it more efficient